modules/ztp-image-based-upgrade.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/ztp-talm-updating-managed-policies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Image-based upgrade for {sno} cluster with the {lcao}
+
+You can upgrade the platform version of a managed {sno} cluster with the {lcao}.
+Using a generated seed image allows you to skip multiple minor releases and it is faster than the standard upgrade method.
+
+You generate the seed image from a seed cluster, which is deployed and configured with the desired {product-title} and applications. You can upgrade the platform version on any {sno} that matches the cluster profile of the seed image.
+
+[IMPORTANT]
+====
+The image-based upgrade uses custom images that are specific to the hardware platform that the clusters are running on.
+Each different hardware platform requires a separate custom image.
+====
+// TODO agree on how much we should specify this. Is this enough or detail about CPU topology, deployment method, etc.
+
+The {lcao} uses two custom resources (CRs):
+
+* The `SeedGenerator` CR specifies the repository to pull the seed image from.
+* The `ImageBasedUpgrade` CR specifies the seed container image for the upgrade of the target cluster and the OADP `Backup` and `Restore` CR configurations to back up your workloads.
```

Comment:
{lcao} runs in two different contexts:

* On the seed cluster, where it will reconcile the SeedGenerator and generate a seed image

* On the cluster-to-be-upgraded, where it will reconcile the ImageBasedUpgrade by pulling a seed image and upgrading from it.

I don't feel these bullets and the line above them communicate that well

Reply:
I rephrased this, PTAL.